backend/src/entity/testEnt.ts

```diff
@@ -0,0 +1,15 @@
+import { Entity, Column, PrimaryGeneratedColumn } from "typeorm"
+
+@Entity()
+export class User {
+    @PrimaryGeneratedColumn()
+    id: number|null
+
+    @Column()
+    name: string
+
+    constructor(name: string) {
```

Comment:
constructor in the entity class???
it's possible but it usually does not make much sense and there is no such example in the doc except mention that if you use constructor all parameters should be optional. I recommend remove constructor at all